Automattic/wp-calypso
# Add Launchpad progress bar component and logic

#### Proposed Changes

* This PR adds the Calypso `<Progressbar />` to the Launchpad sidebar and the logic for calculating the progress bar percentage.

The calculation logic is:  ( Number of completed tasks / Number of total tasks)

<img width="461" alt="image" src="https://user-images.githubusercontent.com/10482592/186178963-68df9906-9394-4de5-b8ae-8ce72912c032.png">


#### Testing Instructions



* Checkout this PR
* Navigate to Launchpad Link in Bio flow - http://calypso.localhost:3000/setup/launchpad?flow=link-in-bio&siteSlug=agrullon95.wordpress.com
* Verify that the progress bar percentage is 60% (3 checklist items completed out of 5 total items)
* Navigate to Launchpad Newsletter flow - http://calypso.localhost:3000/setup/launchpad?flow=newsletter&siteSlug=agrullon95.wordpress.com
* Verify that the progress bar percentage is 75% (3 checklist items completed out of 4 total items)

#### Pre-merge Checklist






Related to #66764
